decidim-participatory_processes/spec/shared/manage_processes_examples.rb

```diff
@@ -31,12 +30,6 @@ def filter_by_group(group_title)
     context "when processes are filtered by process_group" do
       before { filter_by_group(translated(process_group.title)) }
 
-      it "allows the user to edit the process_group" do
```

Comment:
This is this link: 

![Screenshot of the process group edit link](https://github.com/decidim/decidim/assets/717367/7a59c1aa-cbd0-4e8f-8b46-e54277602a14)

As with the Filterable API we don't have this feature, I've checked it out with @carolromero and we agreed to remove this feature